src/libs/ReportActionsUtils.js

```diff
@@ -409,6 +422,49 @@ function getLinkedTransactionID(reportID, reportActionID) {
     return reportAction.originalMessage.IOUTransactionID;
 }
 
+/**
+ * @returns {string}
+ */
+function getMostRecentReportActionLastModified() {
+    // Start with the oldest date possible
+    let mostRecentReportActionLastModified = new Date(0).toISOString();
+
+    // Flatten all the actions
+    // Loop over them all to find the one that is the most recent
+    const flatReportActions = _.flatten(_.map(allReportActions, (actions) => _.values(actions)));
```

Comment:
qq, do you think we'll benefit by using this as:

```
lodash.flatMap(allReportActions, actions => lodash.values(actions));
```

Reply:
To avoid additional iteration, I think we can ignore this one, I am unsure of the performance.